platform/darwin/src/MGLTileSource.h

```diff
@@ -141,6 +142,37 @@ typedef NS_ENUM(NSUInteger, MGLTileCoordinateSystem) {
     MGLTileCoordinateSystemTMS
 };
 
+
+/**
+ An `NSNumber` object containing an unsigned integer that specifies the encoding
+ formula for raster-dem tilesets. The integer corresponds to one of
+ the constants described in `MGLDEMEncoding`.
+
+ The default value for this option is `MGLDEMEncodingMapbox`.
+
+ This option is not supported by the TileJSON spec.
+ */
+extern MGL_EXPORT const MGLTileSourceOption MGLTileSourceOptionDEMEncoding;
+
+/**
+ The encoding formula used to generate the raster-dem tileset
+*/
+
+typedef NS_ENUM(NSUInteger, MGLDEMEncoding) {
+
+    /**
+     Raster tiles generated with the Mapbox encoding formula where
+     elevaion = -10000 + ((R * 256 * 256 + G * 256 + B) * 0.1)
```

Comment:
Typo: `elevation`.

Also, I’m not sure how this comment will be interpreted by reStructuredText and Markdown parsers, but this will format it like a pretty formula:

```html
<var>elevation</var> = −10000 + ((<var>red</var> × 256² + <var>green</var> × 256 + <var>blue</var>))
```